README.md

```diff
@@ -630,10 +682,16 @@ When comparing to `null`, we always compare explicitly.
 ```php
 <?php
 
-function func(ValueClass $value = null)
+function func(?ValueClass $value = null): ?string
```

Comment:
I think that declaring it as ?class $value = null would be the better format, if that is the only argument, and we expect it to be nullable. Because providing null as mandatory argument feels tedious